VAMobile/src/store/slices/secureMessagingSlice.ts

```diff
@@ -184,9 +184,8 @@ export const getInbox =
       //TODO what is the right refersh logic to ensure we don't invoke the API too frequently
       const folderID = SecureMessagingSystemFolderIdConstants.INBOX
       const inbox = await api.get<SecureMessagingFolderGetData>(`/v0/messaging/health/folders/${folderID}`)
-
-      if (getState().secureMessaging.inboxFirstRetrieval && inbox?.data?.attributes?.unreadCount) {
-        await logAnalyticsEvent(Events.vama_hs_sm_count(inbox?.data.attributes.unreadCount))
+      if (getState().secureMessaging.inboxFirstRetrieval && inbox?.data?.attributes) {
+        await logAnalyticsEvent(Events.vama_hs_sm_count(inbox.data.attributes.unreadCount))
```

Comment:
Minor point, but I don't think we ever need to await logAnalyticsEvent, because we don't need to wait for the result. We can just continue code execution while the event gets logged asynchronously